index.js

```diff
@@ -831,6 +906,15 @@ module.exports.bool = {
   or: 'or',
   eor: 'eor'
 };
+// Colourspaces
+module.exports.colourspace = {
+  multiband: 'multiband',
+  'b-w': 'b-w',
+  bw: 'b-w',
+  cmyk: 'cmyk',
+  srgb: 'srgb'
+};
```

Comment:
The docs suggest other colourspaces are available, e.g. `lab`. Should we include them all?